checkNodeVersion.js

```diff
@@ -0,0 +1,21 @@
+const result = process.versions
+if (result && result.node) {
+  if (parseInt(result.node) >= 14) {
+    console.log(
```

Comment:
I don't think we need this. The installation will just proceed if the version meets requirements.